src/components/CloudServiceCard.js

```diff
@@ -0,0 +1,29 @@
+function cloudServiceCard(props){
+    const{title,image}=props.Data;
+    
+    return(
+    <div className="wrapper-div  sm:w-fit md:w-1/5 md:ml-5 mt-5  rounded-lg  shadow-sm border-2 border-x-red-300 shadow-red-300 cursor-pointer hover:shadow-lg " style={props.ParentBg}>
+        <div className="row-span-2 font-[Poppins]">
+            <div className=" md:p-10 p-5 rounded-t-lg ">
+                <img src={image}  alt="aws" className="ml-auto mr-auto w-10 h-10"/>
+                <p className="text-center" style={props.style}>{title} </p>
+            </div>
+            <div className=" md:p-10 p-5  rounded-t-lg" style={props.ChildBg}>
+             <ul className=" list-item">
+                  {
+                  props.Services.map(data => (  
```

Comment:
It will be good idea to check if `props.Services` exists then run the map function
Something like
`props.Service && props.Service.map(data =>())`

Reply:
ok ,i will try to update it